### PR TITLE
Retain authentication on PrivateRoute page reload

### PR DESCRIPTION
**Pull request by PaulMEdwards**

### CODE REVIEW

**Comment by PaulMEdwards** [comment]
[Store token to localStorage on auth](https://trello.com/c/vifYZWsj/41-store-token-to-localstorage-on-auth)

File: src/components/Dashboard.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ class Dashboard extends React.Component {
     axios.defaults.xsrfCookieName = 'csrftoken';
     axios.defaults.headers = {
       'Content-Type': 'application/json',
-      Authorization: `Token ${this.props.authToken}`,
+      Authorization: `Bearer ${this.props.authToken}`,
```
Comment (PaulMEdwards) [inline src/components/Dashboard.js:19]: Turns out this was the main problem... `Token` is not correct here. Needs to be `Bearer`.

File: src/components/Booking.js [hunk 2]
```diff
@@ -56,10 +57,10 @@ class Booking extends React.Component {
     axios.defaults.xsrfCookieName = 'csrftoken';
     axios.defaults.headers = {
       'Content-Type': 'application/json',
-      Authorization: `Token ${this.props.token}`,
+      Authorization: `Bearer ${this.props.authToken}`,
```
Comment (PaulMEdwards) [inline src/components/Booking.js:60]: Found the same problem in the Booking file as well, so fixed that and also the hardcoded localhost API request address at the same time.

File: src/components/Dashboard.js (hunk 1 above)
Comment (danielterry2121) [reply src/components/Dashboard.js:19]: Great catch!

**Review by danielterry2121 — APPROVED** [review]
Great catch on the bearer, everything looks good to me . Excited to have authentication that actually persist !

**Review by Brodt7258 — APPROVED** [review]
looks good, nice job spotting that auth header issue

File: src/components/Booking.js [hunk 3]
```diff
@@ -169,10 +170,10 @@ class Booking extends React.Component {
...
     };
     axios
-      .post('http://127.0.0.1:8000/api/book/', null, {
+      .post(`${actions.PYTHON_API}v1/book/`, null, {
```
Comment (tinomen) [inline src/components/Booking.js:176]: This is happening often, lets wrap this up into a function Yay!

**Pull request merged**